test/eviction/evictpod.go

```diff
@@ -45,22 +45,23 @@ func main() {
 	namespace := flag.String("namespace", "default", "Namespace (defaults to `default`)")
 	pod := flag.String("pod", "", "Pod name (required)")
 	flag.Parse()
+	logger := runtime.NewLoggerWithSource("main")
```

Comment:
```suggestion
	logger := runtime.NewLoggerWithSource("evictpod")
```

Minor of minor tweak (since the file is "evictpod" not "main"), but otherwise, this looks good to go for me.